index.html

```diff
@@ -61,6 +61,165 @@ <h1 class="intro primary-text1">
                 </li>
             </ul>
         </section>
+        <section id="works">
+            <div class="card">
+                <img class="profile-image" src="images/images.png" alt="">
```

Comment:
- [ ] Your image here is broken. Do well to correct that. Also, it is a good practice to always add an alternate so that in case the image is broken or there is a slow connection, the `alt` can explain to the user what is supposed to be there.
- Your design:
![b1](https://user-images.githubusercontent.com/85113567/148500234-13083356-0374-4a7c-b2cd-82290e11442b.jpg)